Docs/Configuration.tex

```diff
@@ -2766,126 +2775,135 @@ \section{Misc}\label{misc}
 \item Mark the option as the default option to boot.
 \item Boot option through the picker or without it depending on the \texttt{ShowPicker} option.
 \item Show picker on failure otherwise.
 \end{itemize}
 
-\emph{Note 1}: This process is meant to work reliably only when \texttt{RequestBootVarRouting}
+\emph{Note 1}: This process will only work reliably when the \texttt{RequestBootVarRouting}
 option is enabled or the firmware does not control UEFI boot options (\texttt{OpenDuetPkg} or
-custom BDS). Without \texttt{LauncherOption} it also is possible that other operating systems
-overwrite OpenCore, make sure to enable it when planning to use them.
+custom BDS). When \texttt{LauncherOption} is not enabled, other operating systems can
+overwrite OpenCore settings and this property should therefore be enabled when planning
+to use other operating systems.
 
-\emph{Note 2}: UEFI variable boot options' boot arguments will be removed if present as they
-may contain arguments compromising the operating system, which is undesired once secure boot
-is enabled.
+\emph{Note 2}: UEFI variable ``boot options'' boot arguments will be removed, if present, as
+they may contain arguments that can compromise the operating system, which is undesirable when
+secure boot is enabled.
 
-\emph{Note 3}: Some operating systems, namely Windows, will create their boot option and
-mark it as top most upon first boot or after NVRAM Reset. When this happens default boot
-entry choice will update till next manual reconfiguration.
+\emph{Note 3}: Some operating systems, such as Windows, may create a boot option and mark it as
+the topmost option upon first boot or after NVRAM resets. When this happens, the default boot
+entry choice will remain changed until the next manual reconfiguration.
 
 \subsection{Properties}\label{miscprops}
 
 \begin{enumerate}
 \item
   \texttt{Boot}\\
   \textbf{Type}: \texttt{plist\ dict}\\
-  \textbf{Description}: Apply boot configuration described in the
+  \textbf{Description}: Apply the boot configuration described in the
   \hyperref[miscbootprops]{Boot Properties} section below.
 
 \item
   \texttt{BlessOverride}\\
   \textbf{Type}: \texttt{plist\ array}\\
-  \textbf{Description}: Add custom scanning paths through bless model.
+  \textbf{Description}: Add custom scanning paths through the bless model.
 
   Designed to be filled with \texttt{plist\ string} entries containing
-  absolute UEFI paths to customised bootloaders, for example,
+  absolute UEFI paths to customised bootloaders such as
   \texttt{\textbackslash EFI\textbackslash debian\textbackslash grubx64.efi}
-  for Debian bootloader. This allows unusual boot paths to be automatically
-  discovered by the boot picker. Designwise they are equivalent to predefined blessed path, such as
+  for the Debian bootloader. This allows non-standard boot paths to be automatically
+  discovered by the boot picker. Designwise, they are equivalent to predefined blessed paths, such as
```

Comment:
Change `Designwise` to `In terms of design` maybe?

Reply:
I like the former more.